profile.go

```diff
@@ -0,0 +1,251 @@
+// Copyright 2021 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package eat
+
+import (
+	"encoding/asn1"
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"strconv"
+	"strings"
+)
+
+const (
+	// ASN1AbsoluteOIDType represents the Type number of Absolute OID ASN Encoding
+	ASN1AbsoluteOIDType = 0x06
+	// ASN1LongLenMask is used to mask bit 8 of Length Indicator byte
+	ASN1LongLenMask = 0x80
+	// ASN1LenBytesMask is used to extract first 7 bits of Length Indicator byte
+	ASN1LenBytesMask = 0x7F
```

Comment:
It doesn't make sense for these three to be part of the public interface.  They should be local to the method that uses them.